translations/translations.json

```
@@ -10159,12 +10159,12 @@
       "appointmentNecessary": "Termin erforderlich",
       "makeAppointmentTooltipWithLink": "Vereinbare an diesem Tag einen Termin über <1>die Webseite</1> oder telefonisch.",
       "theWebsite": "die Webseite",
-      "nearYou_zero": "0 Standorte {{nearYou}}",
-      "nearYou_one": "1 Standort {{nearYou}}",
-      "nearYou_two": "{{count}} Standorte {{nearYou}}",
-      "nearYou_few": "{{count}} Standorte {{nearYou}}",
-      "nearYou_many": "{{count}} Standorte {{nearYou}}",
-      "nearYou_other": "{{count}} Standorte {{nearYou}}"
+      "nearYou_zero": "0 Orte",
+      "nearYou_one": "1 Ort",
+      "nearYou_two": "{{count}} Orte",
+      "nearYou_few": "{{count}} Orte",
+      "nearYou_many": "{{count}} Orte",
+      "nearYou_other": "{{count}} Orte"
```

:wrench: The name does not really fit anymore:
```suggestion
      "poisCount_zero": "0 Orte",
      "poisCount_one": "1 Ort",
      "poisCount_two": "{{count}} Orte",
      "poisCount_few": "{{count}} Orte",
      "poisCount_many": "{{count}} Orte",
      "poisCount_other": "{{count}} Orte"
```